nixos/tests/all-tests.nix

```diff
@@ -892,6 +892,7 @@ in {
   rsyncd = handleTest ./rsyncd.nix {};
   rsyslogd = handleTest ./rsyslogd.nix {};
   rtorrent = handleTest ./rtorrent.nix {};
+  rustls-libssl = handleTest ./rustls-libssl.nix {};
```

Comment:
You could add this to passthru.tests of the package, then ofborg will build it automatically :)

Reply:
Good point. Ofborg ran the test successfully. 👍 
https://logs.ofborg.org/?key=nixos/nixpkgs.363932&attempt_id=c521ecbe-c990-45d8-9984-390fc7331ba3